python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -363,6 +375,37 @@ def analyze(self, event):
                 jets_msdcorr_jmrDown.append(jet_pt_jerNomVal *jet_msdcorr_jmrDownVal*jmsNomVal  *jet_msdcorr_raw)
                 jets_msdcorr_jmsUp  .append(jet_pt_jerNomVal *jet_msdcorr_jmrNomVal *jmsUpVal   *jet_msdcorr_raw)
                 jets_msdcorr_jmsDown.append(jet_pt_jerNomVal *jet_msdcorr_jmrNomVal *jmsDownVal *jet_msdcorr_raw)
+
+                #Also evaluated JMS&JMR SD corr in tau21DDT region: https://twiki.cern.ch/twiki/bin/viewauth/CMS/JetWtagging#tau21DDT_0_43
+                tmp_jmsVals = self.jmsVals
```

Comment:
This is ok, but it looks like it isn't necessary to assign and restore to jmsVals/jmrVals.  What is written only makes use of the values locally here.